source/Patches/EndGameSummary.cs

```diff
@@ -0,0 +1,218 @@
+                    }
+                }
+
+                PluginSingleton<TownOfUs>.Instance.Log.LogMessage($"TaskDone:- " + tasksdone);
+                PluginSingleton<TownOfUs>.Instance.Log.LogMessage($"TaskTotal:- " + taskstotal);
+                pinfo.TasksCompleted = tasksdone;
+                pinfo.TasksTotal = taskstotal;
+                pinfolist.Add(pinfo);
+                tasksdone = 0;
+                taskstotal = 0;
+                AdditionalTempData.playerData = pinfolist;
+            }
+        }
+
+        public static bool HasTasks(Faction faction)
+        {
+            if (faction == Faction.Crewmates)
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        static bool IsNull(AdditionalTempData.PlayerInfo pinfo, string type)
+        {
+            if (type == "Role")
+            {
+                if (pinfo.CurrentRole != null)
+                {
+                    return false;
+                }
+                else
+                {
+                    return true;
+                }
+            }
+            else if (type == "Mod")
+            {
+                if (pinfo.Modifier != null)
+                {
+                    return false;
+                }
+                else
+                {
+                    return true;
+                }
+            }
+            else
+            {
+                return true;
+            }
+
+        }
+
+        public static void LoadGameSummary(EndGameManager __instance)
+        {
+            var position = Camera.main.ViewportToWorldPoint(new Vector3(0f, 1f, Camera.main.nearClipPlane));
+            GameObject roleSummary = UnityEngine.Object.Instantiate(__instance.WinText.gameObject);
+            roleSummary.transform.position = new Vector3(__instance.ExitButton.transform.position.x + 0.1f, position.y - 0.1f, -14f);
+            roleSummary.transform.localScale = new Vector3(1f, 1f, 1f);
+
+            var roleSummaryText = new StringBuilder();
+
+            roleSummaryText.AppendLine("End-of-game Summary:");
+
+            foreach (var data in AdditionalTempData.playerData)
+            {
+                var won = "   ";
+                foreach (var winner in TempData.winners)
+                {
+                    if (winner.Name == data.PlayerName)
+                    {
+                        won = $"[<color=#FAD934FF>W</color>]";
+                    }
+                }
+                var nameinfo = "";
+
+                nameinfo = $"{data.PlayerName}";
+
+
+                var taskInfo = "";//GameData.Instance.AllPlayers[data.PlayerId].PlayerName;
+                                  //if (HasTasks(data.Faction))
+                                  //{
+                taskInfo = data.Faction > 0 ? $"" : $" <color=#FAD934FF>({data.TasksCompleted}/{data.TasksTotal})</color>";
+                //}
```

Comment:
```cs
public static bool HasTasks(Faction faction)
```
What is this for, then?
```suggestion
                var taskInfo = HasTasks(data.Faction) ? "" : $" <color=#FAD934FF>({data.TasksCompleted}/{data.TasksTotal})";
```

Reply:
well.. this will be just me borking up my own code ;)

i was basically having multiple issues so commented out the function to try something else, which then worked, so i left the function commented out. thank you for correcting.

Reply:
your commit suggestion actually removed the </color> required at the end of the line, i will re-add this in with another commit.